replaceNamespace.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+export LANG=C
+export LC_CTYPE=C
+find ./ -type f -exec sed -i '' 's/'"$1"'/'"$2"'/g' {} \;
```

Comment:
please add documentation on the readme.
Why is $1 not a contact (REPLACE_THIS_WITH_NAMESPACE)?